l10n/en/firefox/whatsnew/whatsnew-fx83.ftl

```diff
@@ -0,0 +1,23 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+### URL: https://www-dev.allizom.org/firefox/83.0/whatsnew/all/
+
+# html page title
+whatsnew83-page-title = What’s new with { -brand-name-firefox }
+
+# Main title
+whatsnew83-main-title = Meet { -brand-name-pocket } &mdash; Our inspired source for news & stories
```

Comment:
Is `&mdash;` okay in this FTL file?

Reply:
Yes. It would be worth adding a comment to explain its usage though. (single-line comments in FTL files are indicated with a single #)

`# The — or mdash is used here like a colon :`